extensions/avro/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: "Apache Avro"
+metadata:
+  keywords:
+  - "avro"
+  guide: "https://quarkus.io/guides/kafka"
```

Comment:
Pointing to the Kafka guide is intentional?

Reply:
Yes, Avro is mostly uses with Kafka. So far it's the only usage we have.